src/client/src/pages/DataView360/View/components/Adoptions.js

```diff
@@ -46,11 +52,21 @@ class Adoptions extends Component {
     createRows(data) {
         const result = _.map(data, (row, index) => {
             const photo = row.Photos[0]
+            let eventDate;
+            if (typeof row.Events === 'undefined' || typeof row.Events.Time === 'undefined' ){
+                eventDate = ''
+            }
+            else{
+                eventDate = moment.unix(row.Events.Time).format("YYYY-MM-DD")
+            }
+
```

Comment:
Same thought here for `eventDate`: 

`const eventDate = (row.Events && row.Events.Time) ? moment.unix(row.Events.Time).format("YYYY-MM-DD") : ''`